src/libcore/fmt/mod.rs

```diff
@@ -609,10 +609,15 @@ pub trait Debug {
 /// println!("The origin is: {}", origin);
 /// ```
 #[rustc_on_unimplemented(
+    on(
+        _Self="std::path::Path",
+        label="`{Self}` cannot be formatted with the default formatter, call `.display()` on it",
```

Comment:
```suggestion
        label="`{Self}` cannot be formatted with the default formatter; call `.display()` on it",
```
While error messages aren't full sentences, I think we'd still prefer to avoid [comma splices](https://en.wikipedia.org/wiki/Comma_splice).